### PR TITLE
Add Initial Linux Support

### PR DESCRIPTION
**Pull request by pwelch**
Adding minimum Linux support so Pantry will converge on Debian and RHEL
based systems.

Relates to Pull Request [#2]

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ platforms:
   - name: macosx-10.10
     driver:
       box: chef/macosx-10.10
+  - name: centos-7.0
```
Comment (jtimberman) [inline .kitchen.yml]: centos-7.1

File: .kitchen.yml [hunk 2]
```diff
@@ -21,6 +21,8 @@ platforms:
...
+  - name: debian-7.7
```
Comment (jtimberman) [inline .kitchen.yml]: debian-8.1

**Comment by jtimberman** [comment]
We should use the latest versions from the bento project:

https://github.com/chef/bento/

Debian 8.1, CentOS 7.1

File: .kitchen.yml (hunk 1 above)
Comment (pwelch) [inline .kitchen.yml]: Changed 0e77423

File: .kitchen.yml (hunk 2 above)
Comment (pwelch) [inline .kitchen.yml]: Changed on 0e77423

**Pull request merged**